### PR TITLE
Add t9k scheduler usage for each example of job and autotune

### PR DESCRIPTION
**Pull request by xyxxxxx**
相应地修改了 README

### CODE REVIEW

File: autotune/hpo-keras/autotune_gpu_t9ksched.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    {
+      "batch_size": {"_type": "choice", "_value": [16, 32, 64, 128]},
+      "learning_rate": {"_type": "choice", "_value": [0.0001, 0.001, 0.01, 0.1]},
+      "conv_channels1": {"_type": "choice", "_value": [16, 32, 64, 128]}
+    }
+  trainingConfig:
+    type: tensorflow
+    scheduler:
+      t9kScheduler:
+        queue:
```
Comment (ppkube) [inline autotune/hpo-keras/autotune_gpu_t9ksched.yaml:23]: 1. use a default queue name `default` 
2. use a non-zero value for `priority`, say `10`

Do the same for all other `scheduler` fields

File: autotune/hpo-keras/README.md [hunk 2]
```diff
@@ -11,21 +11,30 @@
...
     ```shell
     # choose one of the following:
-    kubectl create -f autotune.yaml      # CPU training
-    kubectl create -f autotune_gpu.yaml  # GPU training
+    # 1. CPU training
+    kubectl create -f autotune.yaml
+    # 2. GPU training
+    kubectl create -f autotune_gpu.yaml
+    # 3. GPU training with all Jobs scheduled by T9k scheduler
+    vim autotune_gpu_t9ksched.yaml && kubectl create -f autotune_gpu_t9ksched.yaml
```
Comment (ppkube) [inline autotune/hpo-keras/README.md:36]: comment out vim command

```bash
# vim autotune_gpu_t9ksched.yaml
kubectl create -f autotune_gpu_t9ksched.yaml
```

Do this for all other `README.md` files

**Review by ppkube — CHANGES_REQUESTED** [review]
need to provide default values for `queue` in all the yaml files

File: autotune/hpo-keras/autotune_gpu_t9ksched.yaml (hunk 1 above)
Comment (xyxxxxx) [reply autotune/hpo-keras/autotune_gpu_t9ksched.yaml]: modified

File: autotune/hpo-keras/README.md (hunk 2 above)
Comment (xyxxxxx) [reply autotune/hpo-keras/README.md]: modified

**Review by ppkube — APPROVED** [review]
LGTM
